<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractArchiveExtraWidget</class>
 <widget class="QWidget" name="ExtractArchiveExtraWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>142</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Extract</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
      <item>
       <widget class="QRadioButton" name="extractAll">
        <property name="text">
         <string>E&amp;xtract all files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="extractSelected">
        <property name="text">
         <string>Ex&amp;tract selected files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Actions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,1">
      <item>
       <widget class="QCheckBox" name="reCreateFolders">
        <property name="text">
         <string>Re-create folders</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="overwriteExisting">
        <property name="text">
         <string>Overwrite existing files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="skipOlder">
        <property name="text">
         <string>Do not extract older files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
